README.md

```diff
@@ -142,8 +143,9 @@ The output hdf5 files contains two datasets:
 </details>
 
 ## Running tests
-```python
-./test.sh
+```
+pip install ".[tests]"
+pytest
```

Comment:
Is there some mechanism that requires the tests to pass before merging a branch into main?

Reply:
Yes, the CI flow will have to pass; which runs `pytest`